posts/heatmaps-clustering/index.qmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "Heatmaps done right"
+subtitle: "A note on clustering distance matrices before plotting. Think sns.clustermap using altair."
+date: "08/03/2022"
+image: "visualization.png"
+categories: [visualisation, how-to]
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .qmd
+      format_name: quarto
+      format_version: '1.0'
+      jupytext_version: 1.13.7
+  kernelspec:
+    display_name: Python 3.8.12 ('heatmap')
+    language: python
+    name: python3
+---
+
+Given a distance matrix, i.e. a symmetric matrix of distances between observations, if the indices of the observations are arbitrary, or more generally there is no variable by which we want to order the observations, then plotting a heatmap using the default ordering (of the indices) is often not very useful. A better approach is to cluster the observations, and use the new ordering. This is what `sns.clustermap` offers, see [docs](https://seaborn.pydata.org/generated/seaborn.clustermap.html).
```

Comment:
I get the same long nonsense in github. i think the correct way to write md is new line for each new sentence with no space in between, which then gets rendered as single paragraph. dunno.

Reply:
Aha ok, would be good to do that as it makes for a painful read otherwise :)

Reply:
done. we should remember to always do it. (i have for DvsG.) also deffer for diffs. but still not perfect, ie can still have long sentences.